buildSrc/src/main/resources/gocd-docker-agent/Dockerfile.agent.ftl

```diff
@@ -90,8 +93,8 @@ COPY --chown=go:root agent-bootstrapper-logback-include.xml agent-launcher-logba
 COPY --chown=root:root dockerd-sudo /etc/sudoers.d/dockerd-sudo
 </#if>
 
-RUN chown -R go:root /go-agent /docker-entrypoint.d /go /godata /docker-entrypoint.sh \
-    && chmod -R g=u /go-agent /docker-entrypoint.d /go /godata /docker-entrypoint.sh
+RUN chown -R go:root /docker-entrypoint.d /go /godata /docker-entrypoint.sh \
```

Comment:
`/docker-entrypoint.d`, `/go` and `/godata` are empty directories **chown/chmod** on empty directories will not add additional filesize.

Reply:
Whereas, **chmod/chown** on `/docker-entrypoint.sh` file will add additional filesize (same as of the size of the `docker-entrypoint.sh`).

There is no workaround/way for this (as of today), as `ADD` command in docker does not support `--chmod`  flag (it supports `--chown`).